docs/Compatibility.md

```diff
@@ -4,156 +4,247 @@ The Ruby Datadog Trace library is open source. See the [dd-trace-rb][1] GitHub r
+
 [29]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#opensearch
+
 [30]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#postgres
+
 [31]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#presto
+
 [33]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#que
+
 [34]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#racecar
+
 [35]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rack
+
 [36]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rails
+
 [37]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rake
+
 [38]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#redis
+
 [39]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#resque
+
 [40]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#rest-client
+
 [41]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#roda
+
 [42]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sequel
+
 [43]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#shoryuken
+
 [44]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sidekiq
+
 [45]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sinatra
+
 [46]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sneakers
+
 [47]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#stripe
+
 [48]: https://docs.datadoghq.com/tracing/trace_collection/dd_libraries/ruby#sucker-punch
 
+### Support Policy
+
+Datadog APM for Ruby is built upon dependencies defined in specific versions of the host operating system, Ruby
+runtime, certain Ruby libraries, and the Datadog Agent/API. When these versions are no longer supported by their
+maintainers, Datadog APM for Ruby limits its support for these as well.
+
+#### Levels of Support
+
+| **Level**                                              | **Support provided**                                                                                                                                                         |
+|--------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| <span id="support-unsupported">Unsupported</span>      | No implementation. [Contact our customer support team for special requests.](https://www.datadoghq.com/support/)                                                             |
+| <span id="support-beta">Beta</span>                    | Initial implementation. May not yet contain all features. Support for new features, bug & security fixes provided on a best-effort basis.                                    |
```

Comment:
> Initial implementation. May not yet contain all features.

Sounds odd to me, as in it looks like an alpha in disguise.

A definition I would have:

- `alpha`: "Checkpoint"/"early testing" phase. Partial featureset *regarding a target* (e.g for vX.Y we target having feature foo bar baz but only foo+bar are implemented), or feature is partially implemented (known bug, known missing corner case/subfeature, known missing support for an otherwise supported runtime or framework)
- `beta`: "Ironing out" phase. Complete featureset *regarding a target* (e.g for vX.Y we have feature foo bar baz), feature is fully implemented (all corner cases addressed, all runtimes/frameworks supported) but they are not battle-tested (there may be bugs, some might even be known and spelled out in release notes, but will be fixed later)
- `rc`:  Qualification phase. Complete featureset regarding a target, implementation has no known bugs, final testing phase to attempt to surface unknown issues. If none found, promoted "as-is" to GA.

Reply:
In the context of Datadog APM, `beta` means a fully featured feature, but missing enough real-world data before labelling it as full release.

Reply:
@lloeki I agree with your descriptions in the engineering sense of how these terms are actually used. I would suggest that this policy does not describe versions (as we'd attach to our releases) but a general purpose label used for general messaging, particularly to those less initiated in the world of developer terminology.

TL;DR different target audience.

Reply:
I understand (and glad we agree on the technical terminology).

That said, I feel like it can create confusion. For the target audience I would fold this under the umbrella `prerelease` term.